lib/hooks/ReturnStatement.js

```diff
@@ -42,7 +42,7 @@ exports.getIndentEdges = function(node, opts) {
   return parentheses ?
     {
       startToken: parentheses.opening,
-      endToken: parentheses.closing
+      endToken: parentheses.closing.prev
```

Comment:
There are a lot of changes like this, adding `.prev` to the edge's `endToken`. Is that caused by the `moonwalk` to `walk` change?